doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -24,8 +24,8 @@ OSSL_CMP_CTX_set_transfer_cb_arg,
 OSSL_CMP_CTX_get_transfer_cb_arg,
 OSSL_CMP_CTX_set1_srvCert,
 OSSL_CMP_CTX_set1_expected_sender,
-OSSL_CMP_CTX_set0_trustedStore,
-OSSL_CMP_CTX_get0_trustedStore,
```

Comment:
You should keep the old names here as well.

Reply:
Done.